runtime/v1/linux/bundle.go

```diff
@@ -46,12 +47,18 @@ func newBundle(id, path, workDir string, spec []byte) (b *bundle, err error) {
 		return nil, err
 	}
 	path = filepath.Join(path, id)
```

Comment:
Actually just move the 

```go
if err := os.Mkdir(path, 0711); err != nil {
 		return nil, err
}
```

To this location and that will handle the stat issues if it exists.  I did the same thing in runtime v2 code

Reply:
Thanks for your review.
Shall we need to add a task state check before task create in https://github.com/containerd/containerd/blob/master/services/tasks/local.go#L180
If the container's task is exist, we should not enter runtime.Create and return an error.